receiver/vcenterreceiver/metrics.go

```diff
@@ -140,6 +141,27 @@ func (v *vcenterMetricScraper) recordVMStats(
 		return
 	}
 	v.mb.RecordVcenterVMCPUUtilizationDataPoint(ts, 100*float64(cpuUsage)/float64(cpuLimit))
+
+	// OverallCpuReadiness is only available in vSphere API 7.0
+	// https://dp-downloads.broadcom.com/api-content/apis/API_VMA_001/8.0U2/html/vim.vm.Summary.QuickStats.html
+	if v.vsphereAPIVersionMeetsMin("7.0.0") {
+		cpuReadiness := vm.Summary.QuickStats.OverallCpuReadiness
+		v.mb.RecordVcenterVMCPUReadinessDataPoint(ts, int64(cpuReadiness))
+	}
+}
+
+func (v *vcenterMetricScraper) vsphereAPIVersionMeetsMin(minVsphereAPIVersion string) bool {
```

Comment:
Minor: it still feels like this function belongs in the client a little more than here. I don't feel strong enough to halt the PR based on this though, so I'll let you decide.